dango/types/src/lending_pool.rs

```diff
@@ -0,0 +1,75 @@
+use std::collections::BTreeSet;
+
+use grug::{Addr, Coins, Denom};
```

Comment:
Our style is use a single `use` statement for all imports. This sometimes needs to be done manually as the formatter isn't smart enough

Reply:
Ah yes I've read that but missed it here.